Impl

junkdna opened the pull request.

junkdna committed:
dockerfile: clean up after install

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
dockerfile: specify correct directory as volume

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
dockerfile: add curl

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
complete rework

- base on python:alpine
- install dependencies through pip
- initialise database
- set new admin token to value of environment variable or generate new token on
  database initialisation
- remove supervisord and run waitress as only process

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

domi3006 commented on supervisord.conf:
```diff
@@ -4,3 +4,4 @@ nodaemon=true
 [program:iota]
 command=waitress-serve --call 'iota:create_app'
 user=iota
+environment=IOTA_INSTANCE_PATH="/var/iota",USER="iota",HOME="$IOTA_INSTANCE_PATH"
```
Brauchen wir das File denn überhaupt noch? Ich glaube eher nicht. Bitte noch im rework Commit entfernen.

junkdna replied:
done

junkdna committed:
Add github workflow

Using example

https://github.com/actions/starter-workflows/blob/c6c0c7b5efb8a6d4121a9e15b3007b6cc794681e/ci/docker-publish.yml

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
readme: add workflow badge

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
workflow: disable running tests for now

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
dockerfile: update to new URL for ota server

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>

junkdna committed:
dockerfile: allow mkdir to fail gracefully

Signed-off-by: Tillmann Heidsieck <theidsieck@leenox.de>
